app/boards/native_sim_64.overlay

```diff
@@ -0,0 +1,69 @@
+// SPDX-FileCopyrightText: 2025 Florian Limberger <flo@purplekraken.com>
+//
+// SPDX-License-Identifier: MPL-2.0
+
+#include <zephyr/dt-bindings/input/input-event-codes.h>
+#include <zephyr/dt-bindings/lvgl/lvgl.h>
+
+&flashcontroller0 {
+	reg = <0x00000000 DT_SIZE_K(1024 * 100)>;
+};
+
+&flash0 {
+	reg = <0x00000000 DT_SIZE_K(1024 * 100)>;
+
+	partitions {
+		compatible = "fixed-partitions";
+		#address-cells = <1>;
+		#size-cells = <1>;
+
+		flashdisk_partition: partition@0 {
+			reg = <0x00000000 DT_SIZE_K(1024 * 100)>;
+		};
+	};
+};
+
+&sdl_dc {
+	compatible = "zephyr,sdl-dc";
+	height = <480>;
```

Comment:
Undecided: the current orientation is a result of missing driver support for portrait for the used display. We could build the emulator in portrait mode to test and emulate the intended format until the driver is fixed

Reply:
I used these values to emulate the real hardware more closely. I can look into rotating the display, but I think this would best be added as a separate feature.